index.html

```diff
@@ -1,289 +1,647 @@
-                    <input class="form-control" type="text" name="name" placeholder="Full name" maxlength="30" required/>
-                    <input class="form-control" type="email" name="email" id="email" placeholder="Email address" required />
-                    <textarea class="form-control" name="message" id="contact-message" placeholder="Write your message here" required maxlength="500"></textarea>
-                    <button class="contact-btn" type="submit">Get It Touch</button>
-                </form>
+
+        </div>
+
+
+        <div class="card-img-container">
+            <img src="cardFourImage.svg" alt="" class="card-img" />
+        </div>
+
+        
+      </div>
+
+
+
+      <!-- single card ends here -->
+
+       <!-- single card start here -->
+       <div class="card first-card">
+
+        <div class="card-img-container">
+            <img src="cardOneImage.svg" alt="" class="card-img" />
+        </div>
+
+        <div class="card-detail-main-container third-card-cont">
+
+            <div class="card-detail-one">
+                <h2 class="project-title">Facebook 360</h2>
+
+                <ul class="workInfo">
+                    <li class="workInfoDetails first-work">
+                      <a class="workInfoDetailsLink firstWorkLink" href="#">FACEBOOK</a>
+                      <img class="counter" src="Counter.svg" alt="" />
+                    </li>
+                    <li class="workInfoDetails">
+                      <a class="workInfoDetailsLink" href="#">Full Stack Dev</a>
+                      <img class="counter" src="Counter.svg" alt="" />
+                    </li>
+                    <li class="workInfoDetails">
+                      <a class="workInfoDetailsLink" href="#">2015</a>
+                    </li>
+                  </ul>
+  
 
             </div>
-            <div class="contact-heading text-center ">
-                <div class="contact-butt-lin"></div>
+
+            <div class="card-detail-two">
+                <p class="details-sec">
+                    Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.
+                  </p>
+
+                  <ul class="tag-btn-container">
+                    <li class="tag-btn">
+                      <button type="submit" class="btn-small">html</button>
+                    </li>
+                    <li class="tag-btn">
+                      <button type="submit" class="btn-small">Ruby on rails</button>
+                    </li>
+                    <li class="tag-btn">
+                      <button type="submit" class="btn-small">css</button>
+                    </li>
+                    <li class="tag-btn">
+                        <button type="submit" class="btn-small">javascript</button>
+                      </li>
+                  </ul>
+
+                  <button type="submit" class="btnNormal">See Project</button>
```

Comment:
- [ ] Please ensure that the styling for the button state interactions are implemented according to the design

![image](https://user-images.githubusercontent.com/43995830/165738406-b9e9d751-7e60-48c1-a331-b4a679833f3a.png)